<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="attackButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>565</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Attack</string>
    </property>
   </widget>
   <widget class="QPushButton" name="specPowerButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>565</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Special power</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jumpButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>565</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Jump</string>
    </property>
   </widget>
   <widget class="QToolButton" name="upgradeButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>565</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Upgrades</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>1281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>57</y>
      <width>41</width>
      <height>501</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>1281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic_player">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>541</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Images/Bee.png</pixmap>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcdNumber_8"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Max Sheilds</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Current Shields</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcdNumber_7"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_pic_enemy">
    <property name="geometry">
     <rect>
      <x>699</x>
      <y>170</y>
      <width>541</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Images/Bee.png</pixmap>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>999</x>
      <y>70</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcdNumber_9"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Max Shields</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Current Shields</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcdNumber_11"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>699</x>
      <y>70</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Current Health</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcdNumber_12"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Max Health</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcdNumber_13"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>410</y>
      <width>501</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="2" column="1">
      <widget class="QLCDNumber" name="lcdNumber_17"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_20">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Accuracy</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcdNumber_18"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_19">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pierce</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_16">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Weapon Power</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcdNumber_14"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_17">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Armor</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLCDNumber" name="lcdNumber_15"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_18">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Engine Level</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLCDNumber" name="lcdNumber_16"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Player ship</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enemy ship</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>410</y>
      <width>501</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Weapon Power</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcdNumber"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Armor</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLCDNumber" name="lcdNumber_3"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Engine Level</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLCDNumber" name="lcdNumber_4"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pierce</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLCDNumber" name="lcdNumber_2"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Accuracy</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcdNumber_10"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_24">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Scrap</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLCDNumber" name="lcdNumber_20"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Current Health</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcdNumber_5"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Max Health</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcdNumber_6"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>6</y>
      <width>601</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_23">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Turn</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_19"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_28">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Zone</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_24"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_29">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Sector</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_25"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionNew_Game"/>
    <addaction name="actionChoose_Class"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionNew_Game">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="actionChoose_Class">
   <property name="text">
    <string>Choose Class</string>
   </property>
  </action>
  <action name="actionChange_Name">
   <property name="text">
    <string>Change Name</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
